manifests/latest/vsphere-7.0u1/vanilla/deploy/vsphere-csi-controller-deployment.yaml

```diff
@@ -38,7 +38,7 @@ spec:
             - mountPath: /csi
               name: socket-dir
         - name: csi-resizer
-          image: quay.io/k8scsi/csi-resizer:v1.0.0-rc2
+          image: quay.io/k8scsi/csi-resizer:v1.0.0
           args:
             - "--v=4"
             - "--csiTimeout=300s"
```

Comment:
This flag is deprecated - https://github.com/kubernetes-csi/external-resizer/blob/release-1.0/CHANGELOG-1.0.md. We need to use `timeout` instead

Reply:
fixed.